DUI3-DX/Converters/Revit/Speckle.Converters.RevitShared/IRevitVersionConversionHelper.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace Speckle.Converters.RevitShared;
+
+public interface IRevitVersionConversionHelper
+{
+  bool IsCurveClosed(DB.NurbSpline nurbsSpline);
+}
```

Comment:
To me, this and a few of the other interfaces seem like YAGNI. I'm curious what you're thinking about them.

Reply:
No this was to allow us to inject a different version for different revit versions and avoid #if everywhere